byoqm/models/code_climate.py

```diff
@@ -21,6 +25,18 @@ def getDesc(self) -> Dict:
         }
         return model
 
+    def save_to_csv(self, path="./output"):
+        file_location = path + "/" + str(date.today()) + ".csv"
+        if not os.path.exists(path):
+            os.mkdir(path)
+        with open(file_location, "w") as file:
+            writer = csv.writer(file)
+            writer.writerow(["Metric", "Value"])
+            writer.writerow(["Argument Count", str(self.argument_count())])
+            writer.writerow(["File Length", str(self.file_length())])
+            writer.writerow(["Method Count", str(self.method_count())])
+            writer.writerow(["Return Statements", str(self.return_statements())])
```

Comment:
I think it makes most sense to iterate over the `.getDesc()` so we don't have to update this by hand

Reply:
i agree - This was just for getting some output that freddy could use for his visualization